### PR TITLE
Updates code of conduct to match design refresh

### PR DESCRIPTION
**Pull request by antonjb**
Updates the page to match the new design refresh, and:
* Makes emergency phone numbers telephone links for quicker access
* Removed broken code of conduct link for JSConf AU

### CODE REVIEW

File: pages/code-of-conduct.tsx [hunk 1]
```diff
@@ -45,97 +46,130 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
...
 
-    <p>You can make a personal report by:</p>
+    <StyledPara>You can make a personal report by:</StyledPara>
 
     <StyledList>
       <li>
-        Contacting a staff member, identified by {props.pageMetadata.conference.Organiser.ShirtColour} event branded
-        t-shirts and organiser name tags
+        Contacting a staff member, identified by {pageMetadata.conference.Organiser.ShirtColour} event branded t-shirts
+        and organiser name tags
```
Comment (antonjb) [inline pages/code-of-conduct.tsx:64]: is saying colour and type of badge the best option here? Perhaps we should give more detail, more options and a photo. Shouldn't rely solely on colour for a description.

File: pages/code-of-conduct.tsx [hunk 2]
```diff
@@ -45,97 +46,130 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
...
-          {props.pageMetadata.conference.ImportantContacts.Police.Details}
-        </a>
+        <SafeLink href={pageMetadata.conference.ImportantContacts.Police.MapUrl}>
+          {pageMetadata.conference.ImportantContacts.Police.Details}
+        </SafeLink>
+        {pageMetadata.conference.ImportantContacts.Police.Phone && (
+          <Fragment>
+            {' '}
+            <strong>Phone:</strong>{' '}
+            <a href={`tel:${pageMetadata.conference.ImportantContacts.Police.Phone}`}>
```
Comment (antonjb) [inline pages/code-of-conduct.tsx:119]: Made emergency contact phone numbers links so they can be called by tapping them on your phone

File: pages/code-of-conduct.tsx (hunk 1 above)
Comment (mzaatar) [reply pages/code-of-conduct.tsx:64]: special lanyards? wearing a cap?

File: pages/code-of-conduct.tsx (hunk 2 above)
Comment (mzaatar) [reply pages/code-of-conduct.tsx:119]: 👍

File: pages/code-of-conduct.tsx (hunk 1 above)
Comment (mzaatar) [reply pages/code-of-conduct.tsx:64]: could we raise it as an issue on your planner board to be discussed in our next committee meeting?

**Pull request merged**